extensions/apns/apns_message_handler.go

```diff
@@ -245,6 +246,29 @@ func (a *APNSMessageHandler) parseKafkaMessage(message interfaces.KafkaMessage)
 	return notification, nil
 }
 
+func (a *APNSMessageHandler) createDedupContentFromPayload(notification *pusherAPNSKafkaMessage) string {
+	if notification.Payload == nil {
+		return "empty-payload"
+	}
+
+	payloadMap, ok := notification.Payload.(map[string]interface{})
```

Comment:
why `map[string]interface{}` is an invalid payload type? The struct field is `interface{}` so I don't think we can assert that if we can't cast to `map[string]interface{}` then its invalid. You can use reflection, or go a bit simpler and check if it's a string as well